201-web-app-ase-create/azuredeploy.json

```diff
@@ -0,0 +1,172 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aseName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the App Service Environment"
+            }
+        },
+        "aseLocation": {
+            "type": "string",
+            "allowedValues": [
+                "Central US",
+                "East US",
+                "East US 2",
+                "North Central US",
+                "South Central US",
+                "West US",
+                "North Europe",
+                "West Europe",
+                "East Asia",
+                "Southeast Asia",
+                "Japan East",
+                "Japan West",
+                "Brazil South",
+                "Australia East",
+                "Australia Southeast",
+                "West India",
+                "Central India",
+                "South India"
+            ],
+            "metadata": {
+                "description": "Location of the App Service Environment"
+            }
+        },
+        "ipSslAddressCount": {
+            "type": "int",
+            "defaultValue": 1,
+            "metadata": {
+                "description": "Number of IP addresses for the IP-SSL address pool."
+            }
+        },
+        "vnetReference": {
```

Comment:
Is this existing VNET? and is this the complete resourceId of the VNET? In that case the parameter should be called existingVnetResourceId